generators/server/templates/src/test/java/package/config/TestContainersSpringContextCustomizerFactory.java.ejs

```diff
@@ -0,0 +1,60 @@
+package <%= packageName %>.config;
```

Comment:
Split a file for Cassandra another for MongoDB, so we can move each to it's own generator later.

Reply:
Not sure it will be good: there will be multiple combination if we use redis, elastic, couchbase or any dev database that won't be h2.

This is the first PR on the topic, but I plan to make many similar ones.